ArgusCore/src/main/java/com/salesforce/dva/argus/service/mq/kafka/Producer.java

```diff
@@ -136,6 +136,7 @@ public Thread newThread(Runnable r) {
      */
     public <T extends Serializable> int enqueue(final String topic, List<T> objects) {
         int messagesBuffered = 0;
+        _logger.trace("enqueueing topic {}, {} objects", topic, objects.size());
```

Comment:
If you still want this, we should make a separate branch and PR against the public repo, then merge that branch into kiwigrid-develop